composer.json

```
@@ -184,6 +188,7 @@
         "tests": [
             "@check-cs",
             "@phpstan",
+            "@rector",
```

And update this here, if you choose to split the rector scripts:
```suggestion
            "@rector:check",
```